KubeArmor/monitor/logUpdate.go

```diff
@@ -524,6 +524,15 @@ func (mon *SystemMonitor) UpdateLogs() {
 				continue
 			}
 
+			if log.ProcessName == "" {
+				switch log.Operation {
+				case "Process":
+					log.ProcessName = strings.Split(log.Resource, " ")[0]
```

Comment:
Do we need to verify log.Resource and log.Source is empty? If it can be empty, this can cause runtime panic.

Reply:
for the cases where process name as well as source/resource is empty. should we drop that event? similar to: https://github.com/kubearmor/KubeArmor/blob/59ac3021b43cf3d05d4c55b6a372dbfde39a10ec/KubeArmor/feeder/feeder.go#L554